docs/Architecture_Decision_Records/Remove_fetch_content.md

```diff
@@ -0,0 +1,25 @@
+# Remove usage of fetch_content for runtime dependencies
```

Comment:
We'd probably better do this in a preliminary PR

Reply:
https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/2036